index.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="utf-8">
+  <title></title>
+</head>
+<body>
+  <section id="projects">
```

Comment:
Nice job adding a template section from which to construct your projects and then append them!